tools/mason/MasonNew.chpl

```diff
@@ -0,0 +1,82 @@
+
+use Path;
+use TOML;
+use Spawn;
+use FileSystem;
+
+config const vcs: bool = true;
+
+proc InitProject(args: [] string) {
+  const name = args[2];
+  var status = -1;
+  if vcs {
+    status = gitInit(name);
+    addGitIgnore(name);
+  }
+  else {
+    status = noGitInit(name);
+  }
+
+  if status != 0 {
+    halt("Mason failed to initialize your project.");
+  }
+  else {
+    makeBasicToml(name);
+    makeProjectFiles(name);
+    writeln("Mason initialized your new project: " + name);
+  }
+}
+
+
+proc gitInit(name: string) : int {
```

Comment:
I'd be tempted to wrap all of the `git` command functions into a helper-module, e.g. `Git.chpl` -- this would make switching over to `libgit` as a dependency easier in the future.